lib/model/gamers.ts

```diff
@@ -12,21 +12,29 @@ import {Gamer} from '../../src/components/gamer/GamerTypes';
 export async function initializeGamer(queryGamer) {
     const API = await ApiWrapper.getInstance();
     const gamer = await API.MWwz(queryGamer.username, queryGamer.platform);
+    const unoData = await API.MWcombatwzdate(queryGamer.username, 0,0, queryGamer.platform);
+    gamer.uno_id = unoData?.matches[0]?.player.uno;
```

Comment:
@fjaxyu  another thing that is incredibly annoying is that the uno id doesn't show up in the `API.MWwz` call. So I query for their most recent game and get the uno id from there.

Reply:
Oooof, that's really frustrating.